Sources/Classes/core/MiniApp.swift

```diff
@@ -8,6 +11,12 @@ public class MiniApp: NSObject {
         realMiniApp.cleanUpKeychain()
     }
 
+    /// Starts configuration and makes sure to load analytics
+    /// Should be called in AppDelegate didLoad when the app starts
+    static func configure() {
+        MiniAppAnalyticsLoader.loadMiniAppAnalytics()
+    }
+
```

Comment:
Can we have in all methods of this class a check that configure has really been called before MiniApp launch?

Reply:
yeah I've put a check in place, if it's called before using any method basically we can have the check in place for the init method so if `MiniApp.configure()` is not called it will throw an error like so:

![Screen Shot 2022-03-25 at 14 20 33](https://user-images.githubusercontent.com/17570451/160059636-e62fab1d-979e-4f1a-b552-5419da661eae.png)

Reply:
fatalError is a little bit violent XD. But if @rleojoseph and @corycaywood are ok with it I'm fine

Reply:
:D yeah but it has the advantage that it's instantly forcing an error if missed to implement so up to you guys